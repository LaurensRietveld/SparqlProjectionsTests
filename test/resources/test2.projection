{
  "entities": [
    {
      "id": ?entity,
      "birthPlace": ?birthPlace,
      "birthDate": ?birthDate,
      "thumbnail": ?placeholder,
      "title": ?label
    }
  ]
}
