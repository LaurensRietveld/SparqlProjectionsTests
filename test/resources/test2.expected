{
  "entities": [
    {
      "id": "http://dbpedia.org/resource/Marcian",
      "birthPlace": null,
      "birthDate": null,
      "thumbnail": "http://commons.wikimedia.org/wiki/Special:FilePath/Solidus_Marcian_RIC_0509.jpg"
      "title": "Marcian"
    },
    {
      "id": "http://dbpedia.org/resource/Bongbong_Marcos",
      "birthPlace": "http://dbpedia.org/resource/Manila",
      "birthDate": "1957-09-13",
      "thumbnail": "http://commons.wikimedia.org/wiki/Special:FilePath/Bongbong_Marcos.jpg",
      "title": "Bongbong Marcos"
    }
  ]
}
